carl/views.py

```diff
@@ -27,6 +27,7 @@ def bootstrap():
             except json.decoder.JSONDecodeError as je:
```

Comment:
Might be nice to wrap this into a function, for importing into views, and a flask CLI

Reply:
Done - better?